src/components/Collection/scss/index.scss

```diff
@@ -0,0 +1,36 @@
+.data-table {
+  display: flex;
+  flex-direction: row;
+
+  &-info {
+    flex: 1 0 50%;
+    max-width: 50%;
+
+    &-box {
+      margin-bottom: 8px;
```

Comment:
For units like margin, padding, you should use the `$spacer` variable instead or use `rem` units to keep things consistent.